builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -531,6 +531,9 @@ def writeServerlessFile(config, environment_logical_id){
 	sh "sed -i -- 's/\${file(deployment-env.yml):providerTimeout}/${config['providerTimeout']}/g' serverless.yml"
 	sh "sed -i -- 's/\${file(deployment-env.yml):securityGroupIds}/${config['securityGroupIds']}/g' serverless.yml"
 	sh "sed -i -- 's/\${file(deployment-env.yml):subnetIds}/${config['subnetIds'] }/g' serverless.yml"
+  sh "sed -i -- 's|\${file(deployment-env.yml):eventScheduleRate}|${config['eventScheduleRate']}|g' serverless.yml"
```

Comment:
Why are these changes required in api build pack? Shouldn't we deal with these in lambda build pack instead?

Reply:
If these are absolutely required, I would suggest removing the section instead of replacing with empty values (like in the case above).

Reply:
Done. Removed the event scheduled as it is not required in API